proj/settings.py

```diff
@@ -11,8 +11,18 @@
 import socket
 from datetime import date
 
+import environ
```

Comment:
Note: I’m getting `ModuleNotFoundError: No module named 'environ'` even after running `script/build`.

Reply:
Huh, odd. If I rebuild and run it's fine. And the module is in `pyproject.toml` so it should be installed. Not quite sure how to resolve this given I can't replicate it.